src/app/fdctl/ready.c

```diff
@@ -0,0 +1,49 @@
+#include "fdctl.h"
+
+#include "run.h"
+
+#include "../../tango/fd_tango.h"
+
+void
+ready_cmd_fn( args_t *         args,
+              config_t * const config ) {
+  (void)args;
+
+  for( ulong i=0; i<config->shmem.workspaces_cnt; i++ ) {
+    workspace_config_t * wksp = &config->shmem.workspaces[i];
+    switch( wksp->kind ) {
+      case wksp_tpu_txn_data:
+      case wksp_quic_verify:
+      case wksp_verify_dedup:
+      case wksp_dedup_pack:
+      case wksp_pack_bank:
+      case wksp_pack_forward:
+      case wksp_bank_shred:
+      case wksp_bank:
+      case wksp_forward:
+        break;
+      case wksp_quic:
+      case wksp_verify:
+      case wksp_dedup:
+      case wksp_pack: {
+        const uchar * pod = workspace_pod_join( config->name, wksp->name, wksp->kind_idx );
+        fd_cnc_t * cnc = fd_cnc_join( fd_wksp_pod_map( pod, "cnc" ) );
+        int first_iter = 1;
+        do {
```

Comment:
I think we normally do `for(;;)` for loops like this, but maybe that's just the main tile loop